packages/babel-cli/src/babel/options.ts

```diff
@@ -192,7 +192,7 @@ export default function parseArgv(args: Array<string>): CmdOptions | null {
   let filenames = commander.args.reduce(function (globbed, input) {
     let files = glob.sync(input);
     if (!files.length) files = [input];
-    Array.prototype.push.apply(globbed, files);
+    globbed.push(...files);
     return globbed;
   }, []);
```

Comment:
This would be cleaner written with `.flatMap` instead of .`reduce`

Reply:
There are probably some other optimizations that may be done in traverse. Probably a separate PR would make more sense.

Reply:
`flatMap` not available in node < 11.0 :(